src/sentry/models/activity.py

```diff
@@ -32,18 +32,26 @@
 
 class ActivityManager(BaseManager["Activity"]):
     def get_activities_for_group(self, group: Group, num: int) -> Sequence[Group]:
+        from sentry.issues.priority import PRIORITY_LEVEL_TO_STR
+
         activities = []
         activity_qs = self.filter(group=group).order_by("-datetime")
 
         if not features.has("projects:issue-priority", group.project):
-            activity_qs.exclude(type=ActivityType.SET_PRIORITY.value)
+            activity_qs = activity_qs.exclude(type=ActivityType.SET_PRIORITY.value)
+
+        priority = (
+            PRIORITY_LEVEL_TO_STR[group.get_event_metadata()["initial_priority"]]
+            if group.get_event_metadata()["initial_priority"]
+            else None
```

Comment:
Before the priority feature is on, do we still want to store priority in the data (as None) or not have data["priority"] in the Activity at all?

Reply:
It doesn't matter, the frontend just throws that part away. So let's make it consistent and always put priority in there (when it's available)